### PR TITLE
Added emtion to Entities & Keywords, Bug fixes

### PR DESCRIPTION
**Pull request by ACLM**
Bug fixes
- JSON button fixes
- Language URL call fix

### CODE REVIEW

**Comment by ACLM** [comment]
Added emotion to Entities and Keywords at Alexis's request. Also did some fixes for the JSON button not working for some results.

File: views/index.ejs [hunk 1]
```diff
@@ -36,7 +36,7 @@
                 </li>
                 <li class="base--li heading-nav--li" role="presentation">
                     <a class="heading-nav--item" href="https://www.ibm.com/smarterplanet/us/en/ibmwatson/developercloud/starter-kits.html" target="_blank" role="menuitem">
-						Starter Kits
+						App Gallery
```
Comment (germanattanasio) [inline views/index.ejs]: @ACLM Why are you changing this?

**Comment by ACLM** [comment]
That should not be changed. Updating branch.

**Comment by germanattanasio** [comment]
@ACLM can you deploy the code into an url and share that url? I think the result table is not aligned.

**Pull request merged**